source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -1407,7 +1406,51 @@ def test_stop_time_with_detour_and_arrival_before_deleted_at_the_end(self):
 
         # Verify disruptions
         disrupts = self.query_region('disruptions?_current_datetime=20120614T080000')
-        assert len(disrupts['disruptions']) == nb_disruptions_before
+        assert len(disrupts['disruptions']) == nb_disruptions_before + 1
+        assert has_the_disruption(disrupts, 'stop_time_with_detour')
+        last_disrupt = disrupts['disruptions'][-1]
+        assert last_disrupt['severity']['effect'] == 'DETOUR'
+
+        # Verify impacted objects
+        assert len(last_disrupt['impacted_objects']) == 1
+        impacted_stops = last_disrupt['impacted_objects'][0]['impacted_stops']
+        assert len(impacted_stops) == 3
+        assert bool(impacted_stops[0]['is_detour']) is False
+        assert impacted_stops[0]['cause'] == 'on time'
+
+        assert bool(impacted_stops[1]['is_detour']) is True
+        assert impacted_stops[1]['cause'] == 'deleted for detour'
+        assert impacted_stops[1]['departure_status'] == 'unchanged'
+        assert impacted_stops[1]['arrival_status'] == 'deleted'
+
+        assert bool(impacted_stops[2]['is_detour']) is True
+        assert impacted_stops[2]['cause'] == 'added for detour'
+        assert impacted_stops[2]['departure_status'] == 'added'
+        assert impacted_stops[2]['arrival_status'] == 'added'
+
+        B_C_query = "journeys?from={from_coord}&to={to_coord}&datetime={datetime}".format(
```

Comment:
minor: you can probably add `&direct_path=none` to filter the non_pt_walk out of responses.